Assets/PatchKit Patcher/Scripts/Debug/DebugMenu.cs

```diff
@@ -0,0 +1,216 @@
+﻿using System;
+using System.Diagnostics;
+using PatchKit.Unity.Patcher;
+using PatchKit.Unity.Patcher.Debug;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace PatchKit_Patcher.Scripts.Debug
```

Comment:
Please name the `namespace` consistent with all of the project scripts.

Reply:
done